web/js/main.js

```diff
@@ -63,10 +63,20 @@ window.onload = () => {
       config.pageLoadTime = parameters.now
         ? util.parseDateUTC(parameters.now) || new Date()
         : new Date();
+
+      config.initialDate = config.pageLoadTime.getUTCHours() < 3
+        ? new Date((config.pageLoadTime).setUTCDate(config.pageLoadTime.getUTCDate() - 1))
```

Comment:
```suggestion
        ? new Date(config.pageLoadTime.setUTCDate(config.pageLoadTime.getUTCDate() - 1))
```

Reply:
I think the main issue is that `config.pageLoadTime` is being mutated here.

Reply:
Yeah good catch, probably want to double check all those dates for that.